readthedocs/projects/tasks.py

```diff
@@ -578,18 +578,21 @@ def run_build(self, docker, record):
 
         if self.build_env.failed:
             self.send_notifications(self.version.pk, self.build['id'])
-            # send build failure status to git Status API
+            # if the build failed, send build failure status to git Status API
+            # to show status report on the providers pull/merge request UI.
```

Comment:
This is still just explaining what the code does. We can just remove these comments, we don't need to explain anything here.